content/en/providers/check-in/sp/_index.md

```diff
@@ -1163,6 +1163,12 @@ Each level is represented by a URI as follows:
 - **High**: Substantial + multi-factor authn (not yet supported, TBD):
   `https://aai.egi.eu/LoA#High`
 
+Moreover, EGI AAI uses the [REFEDS Assurance framework](https://wiki.refeds.org/display/ASS/REFEDS+Assurance+Framework+ver+1.0).
```

Comment:
Moreover, EGI AAI --> Moreover, EGI Check-in follows the [REFEDS...

Reply:
Done